mergify_engine/queue/merge_train.py

```diff
@@ -1258,21 +1258,30 @@ async def update_summaries(
         if self.has_timed_out:
             conclusion = check_api.Conclusion.FAILURE
 
-        report = check_api.Result(
-            conclusion,
-            title=original_pull_title,
-            summary=unexpected_change_summary
+        check_api_summary = (
+            unexpected_change_summary
             + queue_summary
             + "\n"
             + checks_copy_summary
             + "\n"
-            + batch_failure_summary,
+            + batch_failure_summary
         )
+
+        report = check_api.Result(
+            conclusion,
+            title=original_pull_title,
+            summary=check_api_summary,
+        )
+
         for original_ctxt in original_ctxts:
             original_ctxt.log.info(
                 "pull request train car status update",
                 conclusion=conclusion.value,
-                report=report,
+                original_pull_title=original_pull_title,
+                checked_pull=checked_pull,
+                unexpected_change_summary=unexpected_change_summary.strip(),
+                queue_summary=queue_summary.strip(),
+                batch_failure_summary=batch_failure_summary.strip(),
```

Comment:
We should replace the `checks_copy_summary` part by something`{check.name: check.state for check in self.last_checks}` to get the same content but without the html.